node/src/BaseClient.ts

```diff
@@ -132,6 +131,9 @@ import {
     createZRevRankWithScore,
     createZScore,
     GeoUnit,
+    createZRandMember,
```

Comment:
If you see in other pr's, we have many pr's that fixes the ordering of imports to be alphabetic ( a linter)
maybe you should have this linter yourself so the imports would be ordered good 🙂

Reply:
Will look into the linter!